compiler/tests/evm_contracts.rs

```diff
@@ -648,6 +648,15 @@ fn checked_arithmetic() {
             &[],
         );
 
+
+        // The I256 bug in isolation
+        harness.test_function(
```

Comment:
This is testing the bug in isolation. Basiscally `i256_min_value * 1` doesn't work and I can't figure out why.